service/src/main/java/org/dice/FactCheck/Corraborative/UIResult/create/IPathFactory.java

```diff
@@ -11,7 +11,7 @@
  * @author Michael R&ouml;der (michael.roeder@uni-paderborn.de)
  *
  */
-public interface PathFactory {
+public interface IPathFactory {
```

Comment:
You like to put `I` in front of interfaces? :grinning: 
That is fine by me :+1:

Reply:
Yes I bring it from my .net background :D